Week3/homework/step3.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
 
 function createBase(base) {
-  // Replace this comment and the next line with your code
-  console.log(base);
+	let sum6 = function (add6) {
+		return base + add6;
+	};
+	return sum6;
```

Comment:
Actually, the base may not be 6. It could be any number. I'd rename the parameter.